common/client-libs/gateway-client/src/error.rs

```diff
@@ -102,8 +102,12 @@ impl fmt::Display for GatewayClientError {
                 write!(f, "gateway returned an error response - {}", err)
             }
             GatewayClientError::UnexpectedResponse => write!(f, "received an unexpected response"),
-            GatewayClientError::NotEnoughBandwidth => {
-                write!(f, "client does not have enough bandwidth")
+            GatewayClientError::NotEnoughBandwidth((line_number, estimated, remaining)) => {
+                write!(
+                    f,
+                    "line: {} - client does not have enough bandwidth: estimated {}, remaining: {}",
```

Comment:
I do like the idea of having the estimated/remaining values in the error, but the `line!` seems more useful during debugging rather than for the final build.

Reply:
You're right, we did that one in Crete, will undo :)